### ISSUE TITLE
Make doc fails due to absent backports packages

### ISSUE THREAD

**Issue by grandrew** [jacebrowning/template-python#164]
I was able to run `make doc` on py2.7 and ubuntu 14.04 with added lines to `Pipfile`:

```
[dev-packages]
...
# Documentation
...
backports-abc = "~=0.5"
"backports.ssl_match_hostname" = "*"
```

otherwise it fails with exceptions

**Comment by joshfriend** [comment]
what version of python2.7 was this with? The bump from 2.7.8 to 2.7.9 had some pretty [significant changes regarding SSL verification](https://www.python.org/dev/peps/pep-0476/)

**Comment by grandrew** [comment]
Seems it's 2.7.6 which I believe is what's shipped with ubuntu 14.04 LTS

```
$ pipenv run python
Python 2.7.6 (default, Oct 26 2016, 20:30:19) 
[GCC 4.8.4] on linux2
```

**Comment by jacebrowning** [comment]
I'm sorry you ran into this, but I think the fix for this is simply to upgrade Python. 16.04 LTS comes with a newer version of Python.
